Tasks/Common/Sanitizer/runTests.ps1

```diff
@@ -0,0 +1,12 @@
+$startLoc = Get-Location
+
+Set-Location $PSScriptRoot/../../..
+
+try {
+    Remove-Item .\_build\Tasks\Common\Sanitizer -Recurse -Force -ErrorAction SilentlyContinue
+    node make build --task AzureFileCopyV5
```

Comment:
Is there a specific reason to build `AzureFileCopyV5` in here?

Reply:
This can be any task which uses Sanitizer as dependency due to how make.js allows to build them.
I wanted to update make.js to build just "Common" in a separate PR to speedup merge of these changes, but can do so here to reduce confusion

Reply:
For simplicity; let's just focus on regex change in this PR.

Reply:
done